TRADEMARK-POLICY.md

```diff
@@ -1,14 +1,14 @@
+Meanwhile, the trademarks for this software – including the "LibreHealth" wordmark and the graphic logo – are kept as private property. Trademarks are different than copyrights, particularly in an open source community such as LibreHealth. Trademarks can only be copied for certain specific purposes ("nominative fair use") and cannot be modified in a way to confuse consumers about the origin of the FOSS software ("confusing similarity"). These trademark law terms are described more completely below.
 
-Examples of FOSS trademarks: Linux, Eclipse, Apache, JBoss, MySQL, Firefox, Mozilla, Jaspersoft, Hadoop, Java, OpenOffice....
+Examples of FOSS trademarks: Linux, Eclipse, Apache, JBoss, MySQL, Firefox, Mozilla, Jaspersoft, Hadoop, Java, OpenOffice etc.
 
-"LibreHealth” is like those trademarks. It is a visible brand associated with high-quality open source software for healthcare. It is a brand that was created by our worldwide community. Our brand represents our collective pride and our reputation. We don't want anyone to misuse or misappropriate our brand. We want the value of our trademarks to accrue to the LibreHealth project and its participants as a whole.
+"LibreHealth" is like those trademarks. It is a visible brand associated with high-quality open source software for healthcare. It is a brand created by our worldwide community. Our brand represents our collective pride and our reputation. We do not want anyone to misuse or misappropriate our brand. We want the value of our trademarks to accrue to LibreHealth and its participants as a whole.
 
-Humanitech Inc. is a non-profit corporation that owns and manages all LibreHealth-related trademarks, service marks, and graphic logos in service of our volunteer community. As a US-based corporation, we have a legal responsibility and the authority to set guidelines for the use of our marks. This Trademark Policy outlines how we use our trademarks and logos to identify LibreHealth software developed and distributed by LibreHealth projects. 
+Humanitech Inc. is a non-profit corporation that owns and manages all LibreHealth-related trademarks, service marks, and graphic logos in service of our volunteer community. As a US-based corporation, we have a legal responsibility and authority to set guidelines for the use of our marks. This Trademark Policy outlines how we use our trademarks and logos to identify software developed and distributed by LibreHealth and its products. 
```

Comment:
I think I understand what you're getting at here, but I think the result doesn't read correctly. What about:

> This Trademark Policy outlines how we use our trademarks and logos to identify software & products developed and distributed by the LibreHealth community.